[FIX] pass source_currency params properly

nerdylocks opened the pull request:
addresses RCNX-84
also, use api.ripple.com as a default

[TEST] update build payment method

also add fixtures

[TASK] add lodash

cdm6a commented on test/build_payment.js:
```diff
@@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
 
   before(function () {
     client = new Client({
+      api: 'http://localhost:5990/',
```
This should be hitting api.ripple.com

wltsmrz replied:
In a test? Why?

nerdylocks replied:
Oh you're looking at an older version of the PR. Have fixed that. @wltsmrz

On Wednesday, March 11, 2015, wltsmrz notifications@github.com wrote:

> In test/build_payment.js
> https://github.com/ripple/ripple-rest-client/pull/1#discussion_r26258157
> :
> 
> > @@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
> > 
> >    before(function () {
> >      client = new Client({
> > -      api: 'http://localhost:5990/',
> 
> In a test? Why?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ripple/ripple-rest-client/pull/1/files#r26258157.

## 

_Abiy Seifu |_ Software Engineer
_Ripple Labs, Inc._
abiy@ripple.com | www.ripple.com

wltsmrz replied:
What did you fix?

nerdylocks replied:
ah, removed http://localhost:5990/ so it tests against api.ripple.com.
updated. thanks for catching it @wltsmrz

On Wed, Mar 11, 2015 at 2:42 PM, wltsmrz notifications@github.com wrote:

> In test/build_payment.js
> https://github.com/ripple/ripple-rest-client/pull/1#discussion_r26258904
> :
> 
> > @@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
> > 
> >    before(function () {
> >      client = new Client({
> > -      api: 'http://localhost:5990/',
> 
> What did you fix?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ripple/ripple-rest-client/pull/1/files#r26258904.

## 

_Abiy Seifu |_ Software Engineer
_Ripple Labs, Inc._
abiy@ripple.com | www.ripple.com

wltsmrz replied:
Why would you use api.ripple.com in a test?

nerdylocks replied:
We're testing agains the api.ripple.com because we want to catch breaks in future ripple-rest releases. In a future pull request, we'll have offline testing to make sure our internal (ripple-rest-client) logics work.

cdm6a commented on index.js:
```diff
@@ -108,22 +109,17 @@ Client.prototype.getAccountBalance = function(callback){
-    }
-    sourceCurrenciesParam.source_currencies = sourceCurrenciesString;
+    sourceCurrenciesString = opts.source_currencies.join(',');
+    sourceCurrenciesString += !_.isEmpty(opts.from_issuer) ? '%20' + opts.from_issuer : '';
```
Can we do encodeURI('+') instead?

cdm6a commented on test/fixtures.js:
```diff
@@ -1,7 +1,73 @@
     source_account_secret: process.env.RIPPLE_ACCOUNT_SECRET
+  },
+  responses: {
+    '1_0_4': {
```
Shouldn't this be 1_4_0?

cdm6a commented on test/fixtures.js:
```diff
@@ -1,7 +1,73 @@
+  },
+  responses: {
+    '1_0_4': {
+      error: {
```
'errors'

cdmcnamara commented:
LGTM.

cdmcnamara commented:
LGTM.

Commit:
[FIX] pass source_currency params properly

addresses RCNX-84
also, use api.ripple.com as a default

[TEST] update build payment method

add fixtures for buildPayment responses

Commit:
[TASK] bump version # to 1.13.2

- add lodash
